### ISSUE TITLE
 Unable to apply changes on device. Error is: Command xcodebuild failed with exit code 65.

### ISSUE THREAD

**Issue by tech-oss** [NativeScript/NativeScript#9075]
I have recently updated my native script cli to 6.2.0 from 5.4.0
I migrated my project using the tns migrate command.

But I am getting the following error Command xcodebuild failed with exit code 65. I am only running ios version

Xcode version 12.2

**Package .json:
{
    "nativescript": {
        "id": "com.atrixdigital.flow",
        "tns-android": {
            "version": "6.0.0"
        },
        "tns-ios": {
            "version": "6.2.0"
        }
    },
    "scripts": {
        "build-ios": "tns build ios --bundle --env.uglify && tns prepare ios --release",
        "build-android": "tns build android --bundle --env.uglify --release --keyStorePath atflow.jks --keyStorePassword 122Elgin18! --keyStoreAlias flowAlias --keyStoreAliasPassword 122Elgin18!"
    },
    "description": "NativeScript Application",
    "license": "SEE LICENSE IN <your-license-filename>",
    "repository": "<fill-your-repository-here>",
    "dependencies": {
        "@angular/animations": "8.0.0",
        "@angular/common": "8.0.0",
        "@angular/compiler": "8.0.0",
        "@angular/core": "8.0.0",
        "@angular/forms": "8.0.0",
        "@angular/http": "8.0.0-beta.10",
        "@angular/platform-browser": "8.0.0",
        "@angular/platform-browser-dynamic": "8.0.0",
        "@angular/router": "8.0.0",
        "moment": "^2.24.0",
        "nativescript-angular": "8.0.2",
        "nativescript-appversion": "^1.4.1",
        "nativescript-bottom-navigation": "^1.5.0",
        "nativescript-camera": "^4.2.0",
        "nativescript-checkbox": "^3.0.3",
        "nativescript-datetimepicker": "^1.2.1",
        "nativescript-directions": "^1.3.0",
        "nativescript-filterable-listpicker": "^2.2.3",
        "nativescript-floatingactionbutton": "^5.0.0",
        "nativescript-fontawesome": "^1.0.0",
        "nativescript-gradient": "^2.0.1",
        "nativescript-iqkeyboardmanager": "^1.4.0",
        "nativescript-loading-indicator": "^2.4.0",
        "nativescript-localstorage": "^2.0.0",
        "nativescript-modal-datetimepicker": "^1.1.12",
        "nativescript-phone": "^1.4.1",
        "nativescript-plugin-firebase": "9.0.2",
        "nativescript-socketio": "^3.2.1",
        "nativescript-theme-core": "~1.0.4",
        "nativescript-toasty": "^1.2.0",
        "nativescript-ui-chart": "5.0.0",
        "nativescript-ui-listview": "7.0.0",
        "nativescript-ui-sidedrawer": "7.0.0",
        "reflect-metadata": "~0.1.8",
        "rxjs": "^6.3.3",
        "rxjs-compat": "^6.3.3",
        "tns-core-modules": "6.0.1",
        "zone.js": "0.9.1"
    },
    "devDependencies": {
        "@angular/compiler-cli": "8.0.0",
        "@nativescript/schematics": "~0.5.0",
        "@ngtools/webpack": "8.0.0",
        "nativescript-dev-webpack": "1.0.1",
        "tns-platform-declarations": "^6.5.1",
        "typescript": "3.4.5"
    },
    "gitHead": "9b6565a4513375a33a59c44262dfc1d818a8205e",
    "readme": "NativeScript Application"
}

Build Log:

Platform ios successfully removed.
macbookpro@MacBooks-MBP flow-app % tns run ios
Searching for devices...
Copying template files...
Platform ios successfully added. v6.2.0
Preparing project...
File change detected. Starting incremental webpack compilation...

webpack is watching the files…

Hash: 55c5038a7f9c36c9a462
Version: webpack 4.27.1
Time: 24322ms
Built at: 11/29/2020 22:00:29
                                            Asset       Size                                          Chunks             Chunk Names
                       app/images/Rectangle 4.png   3.88 KiB                                                  [emitted]  
                     app/images/action-select.png   4.17 KiB                                                  [emitted]  
                          app/images/add-card.png    2.3 KiB                                                  [emitted]  
                         app/images/addImage2.png   13.9 KiB                                                  [emitted]  
                          app/images/add_more.png     11 KiB                                                  [emitted]  
                          app/images/addimage.png   6.48 KiB                                                  [emitted]  
                             app/images/alert.png   2.22 KiB                                                  [emitted]  
                      app/images/applications.png   1.24 KiB                                                  [emitted]  
                       app/images/arrow-right.png  480 bytes                                                  [emitted]  
                         app/images/back-icon.png   1.17 KiB                                                  [emitted]  
                              app/images/back.png   12.7 KiB                                                  [emitted]  
                              app/images/bell.png   2.58 KiB                                                  [emitted]  
                             app/images/bell2.png   6.61 KiB                                                  [emitted]  
                             app/images/bell3.png   6.85 KiB                                                  [emitted]  
                              app/images/call.png   1.28 KiB                                                  [emitted]  
                            app/images/camera.png   4.06 KiB                                                  [emitted]  
                 app/images/check-mark-button.png  576 bytes                                                  [emitted]  
                  app/images/checked-inactive.png   1.19 KiB                                                  [emitted]  
                     app/images/checked-input.png   2.13 KiB                                                  [emitted]  
                       app/images/checked-new.png   1.99 KiB                                                  [emitted]  
                           app/images/checked.png   1.41 KiB                                                  [emitted]  
                       app/images/checked_top.png   2.69 KiB                                                  [emitted]  
                          app/images/circle_1.png    2.2 KiB                                                  [emitted]  
                     app/images/crm_icons/TBS.png   2.49 KiB                                                  [emitted]  
                   app/images/crm_icons/arrow.png   2.49 KiB                                                  [emitted]  
               app/images/crm_icons/attention.png   1.84 KiB                                                  [emitted]  
          app/images/crm_icons/back_arrow_crm.png   1.62 KiB                                                  [emitted]  
                  app/images/crm_icons/camera.png   2.51 KiB                                                  [emitted]  
               app/images/crm_icons/completed.png   2.24 KiB                                                  [emitted]  
                app/images/crm_icons/declined.png   2.99 KiB                                                  [emitted]  
       app/images/crm_icons/dropdown_icon_CRM.png   5.65 KiB                                                  [emitted]  
                   app/images/crm_icons/reply.png   1.55 KiB                                                  [emitted]  
         app/images/crm_icons/search_icon_CRM.png   2.46 KiB                                                  [emitted]  
            app/images/crm_icons/underprocess.png   1.98 KiB                                                  [emitted]  
                       app/images/customerotp.png   8.17 KiB                                                  [emitted]  
                     app/images/delete-button.png   1.01 KiB                                                  [emitted]  
                            app/images/delete.png   7.99 KiB                                                  [emitted]  
                        app/images/doc-upload.png   3.65 KiB                                                  [emitted]  
                          app/images/download.jpg    111 KiB                                                  [emitted]  
                               app/images/dsk.png   2.97 KiB                                                  [emitted]  
                         app/images/edit-flat.png   3.87 KiB                                                  [emitted]  
                         app/images/edit-icon.png   19.2 KiB                                                  [emitted]  
                              app/images/edit.png   1.88 KiB                                                  [emitted]  
                              app/images/eida.png    330 KiB                                                  [emitted]  
                        app/images/eida_image.png    566 KiB                                                  [emitted]  
                          app/images/end_task.png   3.96 KiB                                                  [emitted]  
                       app/images/fingerprint.png   32.6 KiB                                                  [emitted]  
                app/images/fingerprint_failed.png   32.1 KiB                                                  [emitted]  
               app/images/fingerprint_success.png   32.7 KiB                                                  [emitted]  
             app/images/floating-btn-inactive.png   12.7 KiB                                                  [emitted]  
                      app/images/floating-btn.png   17.7 KiB                                                  [emitted]  
                              app/images/flow.png   59.3 KiB                                                  [emitted]  
                             app/images/flow2.png   49.8 KiB                                                  [emitted]  
                         app/images/hamburger.jpg    124 KiB                                                  [emitted]  
                         app/images/hamburger.png   4.65 KiB                                                  [emitted]  
                        app/images/hamburger2.png  144 bytes                                                  [emitted]  
                        app/images/hamburger3.png  929 bytes                                                  [emitted]  
                               app/images/ica.jpg   23.2 KiB                                                  [emitted]  
            app/images/icons/account-inactive.png   3.99 KiB                                                  [emitted]  
                     app/images/icons/account.png   2.54 KiB                                                  [emitted]  
                    app/images/icons/bg_color.png     19 KiB                                                  [emitted]  
        app/images/icons/credit-card-inactive.png  942 bytes                                                  [emitted]  
                 app/images/icons/credit-card.png  845 bytes                                                  [emitted]  
              app/images/icons/loans-inactive.png   3.38 KiB                                                  [emitted]  
                       app/images/icons/loans.png   2.93 KiB                                                  [emitted]  
                     app/images/icons/pending.png   5.53 KiB                                                  [emitted]  
                         app/images/info-icon.png   1.18 KiB                                                  [emitted]  
                          app/images/iphone-x.png    146 KiB                                                  [emitted]  
                             app/images/leads.png   4.57 KiB                                                  [emitted]  
                              app/images/logo.png   40.5 KiB                                                  [emitted]  
                            app/images/logout.png   18.5 KiB                                                  [emitted]  
                               app/images/man.png     24 KiB                                                  [emitted]  
                              app/images/menu.png  526 bytes                                                  [emitted]  
                          app/images/navigate.png   1.06 KiB                                                  [emitted]  
                    app/images/new-check-mark.png  835 bytes                                                  [emitted]  
                   app/images/newtouae-active.png   4.95 KiB                                                  [emitted]  
                          app/images/newtouae.png   4.31 KiB                                                  [emitted]  
               app/images/non-resident-active.png   8.14 KiB                                                  [emitted]  
                      app/images/non-resident.png   7.02 KiB                                                  [emitted]  
                      app/images/not-selected.png   1.66 KiB                                                  [emitted]  
                           app/images/pending.png   15.5 KiB                                                  [emitted]  
                          app/images/pending2.png   20.8 KiB                                                  [emitted]  
                            app/images/reader.png    214 KiB                                                  [emitted]  
                   app/images/resident-active.png   6.41 KiB                                                  [emitted]  
                          app/images/resident.png    5.6 KiB                                                  [emitted]  
                   app/images/right-arrow-new.png  731 bytes                                                  [emitted]  
                       app/images/right-arrow.png  965 bytes                                                  [emitted]  
                          app/images/selected.png   1.58 KiB                                                  [emitted]  
                         app/images/selected2.png   20.6 KiB                                                  [emitted]  
                        app/images/start_task.png   4.13 KiB                                                  [emitted]  
                       app/images/take-camera.png   6.28 KiB                                                  [emitted]  
                             app/images/terms.png   3.83 KiB                                                  [emitted]  
                              app/images/user.png   46.4 KiB                                                  [emitted]  
                    app/images/user_profile_2.png   8.42 KiB                                                  [emitted]  
                                        bundle.js   14.1 KiB                                          bundle  [emitted]  bundle
                           fonts/Avenir-Heavy.ttf    131 KiB                                                  [emitted]  
                           fonts/Avenir-Light.ttf   83.1 KiB                                                  [emitted]  
                          fonts/Avenir-Medium.ttf    136 KiB                                                  [emitted]  
                                 fonts/Avenir.ttf     40 KiB                                                  [emitted]  
                     fonts/Proxima Nova Light.ttf   87.7 KiB                                                  [emitted]  
                          fonts/RobotoRegular.ttf    123 KiB                                                  [emitted]  
                    fonts/fontawesome-webfont.ttf    162 KiB                                                  [emitted]  
                                     package.json   81 bytes                                                  [emitted]  
                                       runtime.js   71.4 KiB                                         runtime  [emitted]  runtime
tns_modules/tns-core-modules/inspector_modules.js  643 bytes  tns_modules/tns-core-modules/inspector_modules  [emitted]  tns_modules/tns-core-modules/inspector_modules
                                        vendor.js   3.88 MiB                                          vendor  [emitted]  vendor
Entrypoint bundle = runtime.js vendor.js bundle.js
Entrypoint tns_modules/tns-core-modules/inspector_modules = runtime.js vendor.js tns_modules/tns-core-modules/inspector_modules.js
[./app.css] 3.87 KiB {bundle} [built]
[./main.ts] 926 bytes {bundle} [built]
[./package.json] 82 bytes {bundle} {tns_modules/tns-core-modules/inspector_modules} [optional] [built]
    + 174 hidden modules

ERROR in node_modules/nativescript-toasty/index.d.ts(6,17): error TS2304: Cannot find name 'Color'.
node_modules/nativescript-toasty/index.d.ts(7,23): error TS2304: Cannot find name 'Color'.
node_modules/nativescript-toasty/index.d.ts(11,14): error TS2304: Cannot find name 'Color'.
node_modules/nativescript-toasty/index.d.ts(12,20): error TS2304: Cannot find name 'Color'.
node_modules/nativescript-toasty/index.d.ts(36,23): error TS2304: Cannot find name 'Color'.
node_modules/nativescript-toasty/index.d.ts(43,29): error TS2304: Cannot find name 'Color'.
src/app/assign_report/assign_report.component.ts(24,4): error TS2554: Expected 2 arguments, but got 1.
src/app/assign_report/assign_report.component.ts(25,4): error TS2554: Expected 2 arguments, but got 1.
src/app/chat/chat.tab.component.ts(23,6): error TS2554: Expected 2 arguments, but got 1.
src/app/chat/chat.tab.component.ts(24,6): error TS2554: Expected 2 arguments, but got 1.
src/app/covid_update/covid_update.component.ts(26,4): error TS2554: Expected 2 arguments, but got 1.
src/app/covid_update/covid_update.component.ts(45,4): error TS2554: Expected 2 arguments, but got 1.
src/app/dashboard/dashboard.component.ts(30,4): error TS2554: Expected 2 arguments, but got 1.
src/app/dashboard/dashboard.component.ts(45,4): error TS2554: Expected 2 arguments, but got 1.
src/app/dashboard/dashboard.component.ts(94,4): error TS2554: Expected 2 arguments, but got 1.
src/app/department_status/department_status.component.ts(43,6): error TS2554: Expected 2 arguments, but got 1.
src/app/department_status/department_status.component.ts(62,6): error TS2554: Expected 2 arguments, but got 1.
src/app/edit_profile/edit_profile.component.ts(24,4): error TS2554: Expected 2 arguments, but got 1.
src/app/edit_profile/edit_profile.component.ts(31,4): error TS2554: Expected 2 arguments, but got 1.
src/app/edit_profile/edit_profile.component.ts(63,4): error TS2554: Expected 2 arguments, but got 1.
src/app/extension/extension.component.ts(26,4): error TS2554: Expected 2 arguments, but got 1.
src/app/extension/extension.component.ts(27,4): error TS2554: Expected 2 arguments, but got 1.
src/app/extension/extension.component.ts(51,4): error TS2554: Expected 2 arguments, but got 1.
src/app/faq/faq.component.ts(21,4): error TS2554: Expected 2 arguments, but got 1.
src/app/faq/faq.component.ts(22,4): error TS2554: Expected 2 arguments, but got 1.
src/app/faq/faq.component.ts(43,4): error TS2554: Expected 2 arguments, but got 1.
src/app/graph/graph.component.ts(28,4): error TS2554: Expected 2 arguments, but got 1.
src/app/graph/graph.component.ts(29,4): error TS2554: Expected 2 arguments, but got 1.
src/app/graph/graph.component.ts(49,4): error TS2554: Expected 2 arguments, but got 1.
src/app/graph/graph.component.ts(52,4): error TS2554: Expected 2 arguments, but got 1.
src/app/login/login.component.ts(24,4): error TS2554: Expected 2 arguments, but got 1.
src/app/login/login.component.ts(31,4): error TS2554: Expected 2 arguments, but got 1.
src/app/performance/performance.component.ts(27,4): error TS2554: Expected 2 arguments, but got 1.
src/app/performance/performance.component.ts(39,4): error TS2554: Expected 2 arguments, but got 1.
src/app/performance/performance.component.ts(71,4): error TS2554: Expected 2 arguments, but got 1.
src/app/privacy/privacy.component.ts(27,4): error TS2554: Expected 2 arguments, but got 1.
src/app/privacy/privacy.component.ts(28,4): error TS2554: Expected 2 arguments, but got 1.
src/app/privacy/privacy.component.ts(53,4): error TS2554: Expected 2 arguments, but got 1.
src/app/register/register.component.ts(22,4): error TS2554: Expected 2 arguments, but got 1.
src/app/register/register.component.ts(23,4): error TS2554: Expected 2 arguments, but got 1.
src/app/report/report.component.ts(27,4): error TS2554: Expected 2 arguments, but got 1.
src/app/report/report.component.ts(28,4): error TS2554: Expected 2 arguments, but got 1.
src/app/report/report.component.ts(54,4): error TS2554: Expected 2 arguments, but got 1.
src/app/speciality/speciality.component.ts(41,4): error TS2554: Expected 2 arguments, but got 1.
src/app/speciality/speciality.component.ts(42,4): error TS2554: Expected 2 arguments, but got 1.
src/app/speciality/speciality.component.ts(72,4): error TS2554: Expected 2 arguments, but got 1.
src/app/submit_report/submit_report.component.ts(28,4): error TS2554: Expected 2 arguments, but got 1.
src/app/submit_report/submit_report.component.ts(29,4): error TS2554: Expected 2 arguments, but got 1.
src/app/submit_report/submit_report.component.ts(69,4): error TS2554: Expected 2 arguments, but got 1.
src/app/submitted_report/submitted_report.component.ts(34,4): error TS2554: Expected 2 arguments, but got 1.
src/app/submitted_report/submitted_report.component.ts(35,4): error TS2554: Expected 2 arguments, but got 1.
src/app/submitted_report/submitted_report.component.ts(37,27): error TS2345: Argument of type '{ read: typeof ElementRef; }' is not assignable to parameter of type '{ read?: any; static: boolean; }'.
  Property 'static' is missing in type '{ read: typeof ElementRef; }' but required in type '{ read?: any; static: boolean; }'.
src/app/submitted_report/submitted_report.component.ts(74,4): error TS2554: Expected 2 arguments, but got 1.
src/app/terms/terms.component.ts(20,4): error TS2554: Expected 2 arguments, but got 1.
src/app/terms/terms.component.ts(21,4): error TS2554: Expected 2 arguments, but got 1.
src/app/terms/terms.component.ts(41,4): error TS2554: Expected 2 arguments, but got 1.

Webpack compilation complete. Watching for file changes.
Webpack build done!
nativescript-plugin-firebase: /Users/macbookpro/Documents/flow-app/platforms/ios/.pluginfirebaseinfo not found, forcing prepare!
nativescript-plugin-firebase: running release build or change in environment detected, forcing prepare!
Installing pods...
Analyzing dependencies
Pre-downloading: `Socket.IO-Client-Swift` from `https://github.com/triniwiz/socket.io-client-swift.git`
Downloading dependencies
Installing BEMCheckBox (1.4.1)
Installing Crashlytics (3.14.0)
Installing Fabric (1.10.2)
Installing Firebase (6.2.0)
Installing FirebaseAnalytics (6.0.1)
Installing FirebaseAnalyticsInterop (1.5.0)
Installing FirebaseAuth (6.1.2)
Installing FirebaseAuthInterop (1.1.0)
Installing FirebaseCore (6.0.2)
Installing FirebaseInstanceID (4.2.7)
Installing FirebaseMessaging (4.0.2)
Installing GTMSessionFetcher (1.5.0)
Installing GoogleAppMeasurement (6.0.1)
Installing GoogleUtilities (6.7.2)
Installing IQKeyboardManager (6.2.1)
Installing MBProgressHUD (1.2.0)
Installing MDFInternationalization (2.0.0)
Installing MDFTextAccessibility (2.0.0)
Installing MNFloatingActionButton (2.0.0)
Installing MaterialComponents (112.1.0)
Installing PromisesObjC (1.2.11)
Installing Protobuf (3.13.0)
Installing Socket.IO-Client-Swift (15.1.0)
Installing Starscream (3.1.1)
Installing Toast (3.1.0)
Installing nanopb (0.3.9011)
Generating Pods project
Integrating client project

[!] Please close any current Xcode sessions and use `flowapp.xcworkspace` for this project from now on.
Pod installation complete! There are 12 dependencies from the Podfile and 26 total pods installed.

[!] Crashlytics has been deprecated in favor of FirebaseCrashlytics

[!] Fabric has been deprecated in favor of FirebaseCrashlytics
Project successfully prepared (ios)
Configure firebase
Add iOS crash logging
Building project...
Xcode build...
error: Multiple commands produce '/Users/macbookpro/Documents/flow-app/platforms/ios/build/Debug-iphonesimulator/flowapp.app/Frameworks/MaterialComponents.framework':
1) Target 'flowapp' (project 'flowapp') has copy command from '/Users/macbookpro/Documents/flow-app/node_modules/tns-core-modules-widgets/platforms/ios/MaterialComponents.framework' to '/Users/macbookpro/Documents/flow-app/platforms/ios/build/Debug-iphonesimulator/flowapp.app/Frameworks/MaterialComponents.framework'
2) That command depends on command in Target 'flowapp' (project 'flowapp'): script phase “[CP] Embed Pods Frameworks”

error: Multiple commands produce '/Users/macbookpro/Documents/flow-app/platforms/ios/build/Debug-iphonesimulator/flowapp.app/Frameworks/MDFInternationalization.framework':
1) Target 'flowapp' (project 'flowapp') has copy command from '/Users/macbookpro/Documents/flow-app/node_modules/tns-core-modules-widgets/platforms/ios/MDFInternationalization.framework' to '/Users/macbookpro/Documents/flow-app/platforms/ios/build/Debug-iphonesimulator/flowapp.app/Frameworks/MDFInternationalization.framework'
2) That command depends on command in Target 'flowapp' (project 'flowapp'): script phase “[CP] Embed Pods Frameworks”

note: Using new build system
note: Building targets in parallel
note: Using codesigning identity override: 
note: Planning build
note: Constructing build description
warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99. (in target 'MDFInternationalization' from project 'Pods')
warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99. (in target 'FirebaseAnalyticsInterop' from project 'Pods')
warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99. (in target 'Fabric' from project 'Pods')
warning: duplicate output file '/Users/macbookpro/Documents/flow-app/platforms/ios/build/Debug-iphonesimulator/flowapp.app/Frameworks/MDFInternationalization.framework' on task: PhaseScriptExecution [CP] Embed Pods Frameworks /Users/macbookpro/Library/Developer/Xcode/DerivedData/flowapp-bvfakajbhhlhcublttytjyenrbsv/Build/Intermediates.noindex/flowapp.build/Debug-iphonesimulator/flowapp.build/Script-F14874CEE3DEF51A0AD9E36B.sh (in target 'flowapp' from project 'flowapp')
warning: duplicate output file '/Users/macbookpro/Documents/flow-app/platforms/ios/build/Debug-iphonesimulator/flowapp.app/Frameworks/MaterialComponents.framework' on task: PhaseScriptExecution [CP] Embed Pods Frameworks /Users/macbookpro/Library/Developer/Xcode/DerivedData/flowapp-bvfakajbhhlhcublttytjyenrbsv/Build/Intermediates.noindex/flowapp.build/Debug-iphonesimulator/flowapp.build/Script-F14874CEE3DEF51A0AD9E36B.sh (in target 'flowapp' from project 'flowapp')
warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99. (in target 'MDFTextAccessibility' from project 'Pods')
warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99. (in target 'GoogleUtilities' from project 'Pods')
warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99. (in target 'IQKeyboardManager' from project 'Pods')
warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99. (in target 'Toast' from project 'Pods')
warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99. (in target 'MBProgressHUD' from project 'Pods')
warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99. (in target 'Crashlytics' from project 'Pods')
warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99. (in target 'MNFloatingActionButton-MNFloatingActionButton' from project 'Pods')
warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99. (in target 'nanopb' from project 'Pods')
warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99. (in target 'Firebase' from project 'Pods')
warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99. (in target 'MNFloatingActionButton' from project 'Pods')
warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99. (in target 'FirebaseCore' from project 'Pods')
warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99. (in target 'Protobuf' from project 'Pods')
warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99. (in target 'FirebaseAuthInterop' from project 'Pods')
warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99. (in target 'Starscream' from project 'Pods')
warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99. (in target 'Socket.IO-Client-Swift' from project 'Pods')
warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99. (in target 'BEMCheckBox' from project 'Pods')
warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99. (in target 'FirebaseInstanceID' from project 'Pods')
warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99. (in target 'FirebaseAuth' from project 'Pods')
warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99. (in target 'FirebaseMessaging' from project 'Pods')
warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99. (in target 'GoogleAppMeasurement' from project 'Pods')
warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99. (in target 'FirebaseAnalytics' from project 'Pods')
** BUILD FAILED **

Unable to apply changes on device: E09830FE-3B2B-4488-B250-5EF17CE0281F. Error is: Command xcodebuild failed with exit code 65.

**Comment by tech-oss** [comment]
@NickIliev Please assist if you have the solution to the problem

**Comment by tech-oss** [comment]
I have tried to run this on xcode 11.2 as well but still getting the same issue. So one thing I am sure about is the xcode version is not causing the error

**Comment by tech-oss** [comment]
Any one there who can assist ASAP

**Comment by tech-oss** [comment]
After updating the plugins using command tns plugin update I am only left with these errors:
error: Multiple commands produce '/Users/macbookpro/Documents/flow-app/platforms/ios/build/Debug-iphonesimulator/flowapp.app/Frameworks/TNSCore.framework':
1) Target 'flowapp' (project 'flowapp') has copy command from '/Users/macbookpro/Documents/flow-app/node_modules/nativescript-ui-chart/node_modules/nativescript-ui-core/platforms/ios/TNSCore.framework' to '/Users/macbookpro/Documents/flow-app/platforms/ios/build/Debug-iphonesimulator/flowapp.app/Frameworks/TNSCore.framework'
2) Target 'flowapp' (project 'flowapp') has copy command from '/Users/macbookpro/Documents/flow-app/node_modules/nativescript-ui-core/platforms/ios/TNSCore.framework' to '/Users/macbookpro/Documents/flow-app/platforms/ios/build/Debug-iphonesimulator/flowapp.app/Frameworks/TNSCore.framework'
3) Target 'flowapp' (project 'flowapp') has copy command from '/Users/macbookpro/Documents/flow-app/node_modules/nativescript-ui-listview/node_modules/nativescript-ui-core/platforms/ios/TNSCore.framework' to '/Users/macbookpro/Documents/flow-app/platforms/ios/build/Debug-iphonesimulator/flowapp.app/Frameworks/TNSCore.framework'

note: Using new build system
note: Planning build
note: Constructing build description
warning: duplicate output file '/Users/macbookpro/Documents/flow-app/platforms/ios/build/Debug-iphonesimulator/flowapp.app/Frameworks/TNSCore.framework' on task: PBXCp /Users/macbookpro/Documents/flow-app/node_modules/nativescript-ui-core/platforms/ios/TNSCore.framework /Users/macbookpro/Documents/flow-app/platforms/ios/build/Debug-iphonesimulator/flowapp.app/Frameworks/TNSCore.framework (in target 'flowapp' from project 'flowapp')
warning: duplicate output file '' on task: PBXCp /Users/macbookpro/Documents/flow-app/node_modules/nativescript-ui-core/platforms/ios/TNSCore.framework /Users/macbookpro/Documents/flow-app/platforms/ios/build/Debug-iphonesimulator/flowapp.app/Frameworks/TNSCore.framework (in target 'flowapp' from project 'flowapp')
warning: duplicate output file '/Users/macbookpro/Documents/flow-app/platforms/ios/build/Debug-iphonesimulator/flowapp.app/Frameworks/TNSCore.framework' on task: PBXCp /Users/macbookpro/Documents/flow-app/node_modules/nativescript-ui-listview/node_modules/nativescript-ui-core/platforms/ios/TNSCore.framework /Users/macbookpro/Documents/flow-app/platforms/ios/build/Debug-iphonesimulator/flowapp.app/Frameworks/TNSCore.framework (in target 'flowapp' from project 'flowapp')
warning: duplicate output file '' on task: PBXCp /Users/macbookpro/Documents/flow-app/node_modules/nativescript-ui-listview/node_modules/nativescript-ui-core/platforms/ios/TNSCore.framework /Users/macbookpro/Documents/flow-app/platforms/ios/build/Debug-iphonesimulator/flowapp.app/Frameworks/TNSCore.framework (in target 'flowapp' from project 'flowapp')
** BUILD FAILED **

Unable to apply changes on device: 4D8959B3-257F-4272-9C0C-D34F4E5C4D82. Error is: Command xcodebuild failed with exit code 65.

**Comment by rigor789** [comment]
XCode 12 needs CLI 7.0.10+

See: https://nativescript.org/blog/nativescript-6-7-xcode-compatibility/

**Comment by tech-oss** [comment]
I have downgraded & now using xcode 11.2 but still same issue

**Comment by shilik** [comment]
Hi,

Here is my working settings for nativescript. Just give it a try and good
luck.

xcode 12.1:
nativescript:6.8.0
tns-ios:6.5.3
tns-core-modules:6.5.21
nativescript-vue and nativescript-vue-template-compiler to ~2.7.1
App_Resources/iOS/
build.xcconfig：
Add：
EXCLUDED_ARCHS__EFFECTIVE_PLATFORM_SUFFIX_simulator__NATIVE_ARCH_64_BIT_x86_64=arm64
arm64e armv7 armv7s armv6 armv8
EXCLUDED_ARCHS=$(inherited)
$(EXCLUDED_ARCHS__EFFECTIVE_PLATFORM_SUFFIX_$(EFFECTIVE_PLATFORM_SUFFIX)__NATIVE_ARCH_64_BIT_$(NATIVE_ARCH_64_BIT))

Add a Podfile with the following contents  in the same directory of
App_Resources/iOS/

post_install do |installer|
installer.pods_project.targets.each do |target|
target.build_configurations.each do |config|
config.build_settings['EXCLUDED_ARCHS[sdk=iphonesimulator*]'] = 'arm64'
config.build_settings.delete 'IPHONEOS_DEPLOYMENT_TARGET'
config.build_settings['CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER'] =
'NO'
end
end
end

**Comment by Upd4ting** [comment]
You need to create a `Podfile` into `app/App_Resources/iOS` with that content:

```
post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
      config.build_settings.delete 'IPHONEOS_DEPLOYMENT_TARGET'
    end
  end
end
```

And if you're using nativescript cli >= 7.0 you need Xcode 12. But be careful with Xcode 12.3 that is having an issue currently, you can know more about that here #9097

**Comment by janoshrubos** [comment]
Closing this issue due to inactivity. 
If this error is still relevant, feel free to open a new issue!

**Comment by adideas** [comment]
HI !

Here is what helped me 
- XCODE 13.3.1
- NativeScript 8.0.2 cli
- Platform IOS "6.5.2"

STEP 1 (build.xcconfig)

```
// xcode 13
EXCLUDED_ARCHS__EFFECTIVE_PLATFORM_SUFFIX_simulator__NATIVE_ARCH_64_BIT_x86_64=arm64 arm64e armv7 armv7s armv6 armv8
EXCLUDED_ARCHS=$(inherited)
$(EXCLUDED_ARCHS__EFFECTIVE_PLATFORM_SUFFIX_$(EFFECTIVE_PLATFORM_SUFFIX)__NATIVE_ARCH_64_BIT_$(NATIVE_ARCH_64_BIT))

IPHONEOS_DEPLOYMENT_TARGET = 12.0
```


STEP 2 (Podfile)

```
pod 'CocoaAsyncSocket'

post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
      config.build_settings.delete 'IPHONEOS_DEPLOYMENT_TARGET'
      config.build_settings['EXCLUDED_ARCHS[sdk=iphonesimulator*]'] = 'arm64'
      config.build_settings['CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER'] ='NO'
      config.build_settings['ONLY_ACTIVE_ARCH'] = 'YES'
    end
  end
end

IPHONEOS_DEPLOYMENT_TARGET = 12.0
```
